core-ui/src/state/navigation/filters/filterScopeNodes.ts

```diff
@@ -0,0 +1,24 @@
+import { NavNode } from '../../types';
+
+export const filterScopeNodes = (
+  navList: NavNode[],
+  scope: 'cluster' | 'namespace',
+) => {
+  if (!(scope === 'cluster' || scope === 'namespace')) {
+    console.error('Navigation scope is not defined.');
+    return [];
+  }
```

Comment:
I don't think this is needed? TS is already complaining if I use a different string for `scope`.